### PR TITLE
Add donut graph for object status. Fixes #246

### LINKED ISSUE #246: Donut graph of active vs inactive objects
Count "suppressed" and not suppressed objects to populate the donut graph on the repository activity page.  See mockup at:

https://ggeisler.github.io/lerna-dashboard/admin.html

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: config/locales/hyrax.es.yml [hunk 1]
```diff
@@ -137,6 +140,11 @@ es:
           heading: "Ver Archivos depositados:"
           start_label: "Comenzar"
         registered: "Registrado"
+        repository_objects:
+          series:
+            published:   'Published'
```
Comment (mjgiarlo) [inline config/locales/hyrax.es.yml]: These look like they need translation.

File: app/search_builders/hyrax/stats/work_status_search_builder.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      # use caution when combining this with other searches as it sets the rows to
+      # zero to just get the facet information
+      # @param solr_parameters the current solr parameters
+      def include_suppressed_facet(solr_parameters)
+        solr_parameters[:"facet.field"].concat([IndexesWorkflow.suppressed_field])
+        solr_parameters[:'facet.missing'] = true
+        # only get work information
+        solr_parameters[:fq] = Hyrax::WorkRelation.new.search_model_clause
+
+        # we only want the facte counts not the actual data
```
Comment (mjgiarlo) [inline app/search_builders/hyrax/stats/work_status_search_builder.rb]: Typo: `facte`

File: app/indexers/hyrax/indexes_workflow.rb [hunk 3]
```diff
@@ -1,6 +1,11 @@
 module Hyrax
   module IndexesWorkflow
     STORED_BOOL = Solrizer::Descriptor.new(:boolean, :stored, :indexed)
+
+    mattr_accessor :suppressed_field, instance_writer: false do
```
Comment (mjgiarlo) [inline app/indexers/hyrax/indexes_workflow.rb]: Can you say why this change is needed?

File: app/views/hyrax/admin/show.html.erb [hunk 4]
```diff
@@ -36,4 +36,20 @@
       </div>
     </div>
 </div>
+
 <%= render 'collections' %>
+
+<div class="row">
```
Comment (mjgiarlo) [inline app/views/hyrax/admin/show.html.erb]: Add a view spec?

File: app/indexers/hyrax/indexes_workflow.rb (hunk 3 above)
Comment (jcoyne) [inline app/indexers/hyrax/indexes_workflow.rb]: So that we can reference the value from outside this module

File: app/views/hyrax/admin/show.html.erb (hunk 4 above)
Comment (jcoyne) [inline app/views/hyrax/admin/show.html.erb]: I don't think I can test this as it goes into a CDATA block and thus isn't parsed by capybara.

File: app/indexers/hyrax/indexes_workflow.rb (hunk 3 above)
Comment (mjgiarlo) [inline app/indexers/hyrax/indexes_workflow.rb]: :ok_hand:

File: app/views/hyrax/admin/show.html.erb (hunk 4 above)
Comment (mjgiarlo) [inline app/views/hyrax/admin/show.html.erb]: Foo.

**Comment by jcoyne** [comment]
@mjgiarlo is there anything further here?

**Comment by mjgiarlo** [comment]
I think this can go in now.
